### PR TITLE
Fixed duplicate timestamps causing pagination test failure

### PR DESCRIPTION
**Pull request by ahossameldin**

### CODE REVIEW

File: db_test.go [hunk 1]
```diff
@@ -1507,7 +1541,11 @@ func TestPagedQueryUsingIndex(t *testing.T) {
 		n := 10003
 		for i := 0; i < n; i++ {
 			doc := d.NewDocument()
-			doc.Set("timestamp", time.Now())
+			// Old code that caused duplicates:
```
Comment (ostafen) [inline db_test.go]: I think this comments here just add noise

File: db_test.go [hunk 2]
```diff
@@ -1507,7 +1541,11 @@ func TestPagedQueryUsingIndex(t *testing.T) {
...
+			// doc.Set("timestamp", time.Now())
+
+			// New code to ensure unique timestamps:
```
Comment (ostafen) [inline db_test.go]: This comment makes sense, but there is no need to mention about what "old code" was doing

File: db_test.go [hunk 3]
```diff
@@ -1494,6 +1494,40 @@ func TestIndexQueryWithSort(t *testing.T) {
 	})
 }
 
+func checkUniqueTimestamps(db *c.DB) {
+	// Retrieve all documents from the collection
```
Comment (ostafen) [inline db_test.go]: Let's remove `checkUniqueTimestamps`, it sounds something useful only for debugging during development.

**Review by ostafen — COMMENTED** [review]
Thanks for the PR, @ahmed-hossam28. Added some comments.

**Comment by ahossameldin** [comment]
Thank you @ostafen  for your efforts too, I will resolve them.

**Comment by ahossameldin** [comment]
@ostafen, I have removed the unnecessary comments and the checkUniqueTimestamps function as suggested.

File: db_test.go [hunk 4]
```diff
@@ -1507,7 +1507,11 @@ func TestPagedQueryUsingIndex(t *testing.T) {
 		n := 10003
 		for i := 0; i < n; i++ {
 			doc := d.NewDocument()
-			doc.Set("timestamp", time.Now())
+
+			// doc.Set("timestamp", time.Now())
```
Comment (ostafen) [inline db_test.go]: Please remove all the comments

File: db_test.go (hunk 2 above)
Comment (ostafen) [reply db_test.go]: You can leave this one, but rephrase it to not mention "old code"

**Review by ostafen — CHANGES_REQUESTED** [review]
Added some more comments

**Comment by ahossameldin** [comment]
Thank you @ostafen for your feedback, i updated the comments.

**Review by ostafen — APPROVED** [review]

**Comment by ostafen** [comment]
Looks good now, thanks for your patience!
